### DISCUSSION TITLE
0.14.2

### DISCUSSION THREAD

**Discussion by navarr** [wardenenv/warden#732] [Announcements]
Welcome to 2024!  Let's start this year off with an exciting Warden release!

## What's Changed
* Project and Local Warden commands will now take precedence over built-in warden commands ([#676](https://github.com/wardenenv/warden/pull/676) by @flpandre)
* New Magento 2 projects will default to the latest version of Magepack ([#712](https://github.com/wardenenv/warden/pull/712) by @sprankhub)
* Traefik will now properly fallback to the default Warden certificate ([#725](https://github.com/wardenenv/warden/pull/725) by @SamJUK)
* `warden vnc` now properly connects to the container ([#721](https://github.com/wardenenv/warden/pull/721) by @sprankhub)
* `warden status` will now properly work with dashes in project names and urls ([#685](https://github.com/wardenenv/warden/pull/685) by @bap14)

## New in Images
Images are regularly updated, and these fixes roll out ASAP after they are merged.  This is just a summary of what changes have occurred between the last release and the new one.

* MagePack Bundle and Generate files have corrected permissions ([images#12](https://github.com/wardenenv/images/pull/12) by @ihor-sviziev)
* Resolved issues with node update crashing the php container
* Updated Nginx mapping to route to XDebug when XDEBUG get/post session parameters are present
* Added versions of services:
  * ElasticSearch 8.9, 8.10, 8.11
  * MariaDB 10.10, 10.11, 11.0, 11.1, 11.2, 11.3-rc
  * MySQL 8.2
  * Nginx 1.24, 1.25
  * OpenSearch 2.7, 2.8, 2.9, 2.10, 2.11
  * PHP 8.3
  * RabbitMQ 3.12, 3.13-rc
  * Redis 7.2
  * Varnish 7.3, 7.4

## New in Documentation
Documentation is regularly updated, and these changes roll out ASAP after they are merged.  This is just a summary of what changes have occurred between the last release and the new one.

* A broken link from Installation to the DNS Resolver configuration page was fixed ([docs#18](https://github.com/wardenenv/docs/pull/18) by @danni140c)
* The Magento 2 testing documentation has been updated to point to the correct config directory when referring to API tests ([docs#11](https://github.com/wardenenv/docs/pull/11) by @evktalo)

## Our Sponsors
A huge shoutout to our Gold sponsors between 0.14.1 and 0.14.2:

[![SwiftOtter](https://warden.dev/img/sponsors/swiftotter.svg)](https://www.swiftotter.com/)  
[![Sansec.io](https://warden.dev/img/sponsors/sansec.svg)](https://www.sansec.io/)  
[![Hyvä](https://user-images.githubusercontent.com/145128/226427529-53483968-c9ab-484a-9ae3-c6abb58f81c9.png)](https://www.hyva.io/)

<hr /><em>This discussion was created from the release <a href='https://github.com/wardenenv/warden/releases/tag/0.14.2'>0.14.2</a>.</em>

**Comment by emreakay** [comment]
how can i upgrade my warden

**Reply by bap14** [reply]
@emreakay Depends on how you installed it. If it was via brew, just run `brew upgrade warden`. If you installed it manually using git, you can just run a `git pull` then `warden svc down && warden svc up` to restart all services. If you installed it manually any other way, you'd have to essentially do whatever you did before, replace the existing installation files with the current version, then restart warden services.

**Reply by emreakay** [reply]
thanks
